src/kem/kyber/sampling.rs

```diff
@@ -37,6 +37,19 @@ pub fn sample_from_uniform_distribution<const SEED_SIZE: usize>(
     (out, Some(BadRejectionSamplingRandomnessError))
 }
 
+#[inline(always)]
+pub(super) fn sample_from_binomial_distribution<const ETA: usize>(
+    randomness: &[u8],
+) -> KyberPolynomialRingElement {
+    debug_assert_eq!(randomness.len(), ETA * 64);
+
+    match ETA {
```

Comment:
This will fail with the current version of hax.

```suggestion
    match ETA as u32 {
```